test/models/thesis_test.rb

```diff
@@ -1245,4 +1245,100 @@ def attach_file_with_purpose_to(thesis, purpose = 'thesis_pdf')
     assert thesis.authors.empty?
     assert multi_author_count - 1, Thesis.multiple_authors.count
   end
+
+  test 'consented_to_proquest returns only theses with authors that have opted in to ProQuest' do
+    proquest_consent_count = Thesis.consented_to_proquest.count
+    assert proquest_consent_count < Thesis.count
+
+    # single-author thesis that has opted in is included
+    thesis = theses(:one)
+    assert_includes Thesis.consented_to_proquest, thesis
+
+    # single-author thesis that has opted out is excluded
+    thesis = theses(:with_note)
+    assert_not_includes Thesis.consented_to_proquest, thesis
+
+    # single-author thesis with no opt-in status is excluded
+    thesis = theses(:coauthor)
+    assert_not_includes Thesis.consented_to_proquest, thesis
+
+    # multi-author thesis that has opted in is included
+    thesis = theses(:two)
+    assert_includes Thesis.consented_to_proquest, thesis
+
+    # multi-author thesis that has opted out is excluded
+    thesis = theses(:with_hold)
+    assert_not_includes Thesis.consented_to_proquest, thesis
+
+    # multi-author thesis with conflicting opt-in statuses is excluded
+    thesis = theses(:doctor)
+    assert_not_includes Thesis.consented_to_proquest, thesis
+  end
+
+  test 'invalid without proquest_exported' do
```

Comment:
This test name may benefit from some clarity. None of the assertions seem to result in `invalid` which I'd expect based on this test name.